person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] Kindly fix this typo error as the function is `of_age?` instead of `is_of_age?`.